Lib/test/libregrtest/save_env.py

```diff
@@ -65,8 +66,40 @@ def __init__(self, testname, verbose=0, quiet=False, *, pgo=False):
                  'sysconfig._CONFIG_VARS', 'sysconfig._INSTALL_SCHEMES',
                  'files', 'locale', 'warnings.showwarning',
                  'shutil_archive_formats', 'shutil_unpack_formats',
+                 'asyncio.get_event_loop_policy', 'asyncio.get_event_loop',
+                 'asyncio_get_exception_handler', 'asyncio_get_debug',
+                 'asyncio_get_child_watcher',
                 )
 
+    def get_asyncio_get_event_loop_policy(self):
+        return asyncio.get_event_loop_policy()
+    def restore_asyncio_get_event_loop_policy(self, policy):
+        asyncio.set_event_loop_policy(policy)
+
+    def get_asyncio_get_event_loop(self):
+        return asyncio.get_event_loop()
```

Comment:
This function has a side effect: if there is no current event loop, it does create a new one. So I don't think that it's safe to call it. I don't think that we have an API to get the event loop without side effect.

Reply:
Then basically the whole PR is pointless as every single thing here fetches the event loop somehow to check its state.